frontend/src/components/appViews/SpecificTeamView.tsx

```diff
@@ -32,6 +41,7 @@ function SpecificTeamView() {
 
 	return (
 		<AppView title={team?.name ?? ''} barComponent={<SpecificTeamViewBar />}>
+			<GoBackButton />
```

Comment:
Skulle vi flytte den op i team-navbaren?

Reply:
Sure mate

Reply:
Så til venstre for navnet på teamet?